addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -74,7 +75,7 @@ def _compute_web_page_link(self):
 
     @api.depends('channel_ids')
     def _compute_nbr_channel(self):
-        data = self.env['mail.channel'].read_group([('livechat_channel_id', 'in', self._ids)], ['__count'], ['livechat_channel_id'], lazy=False)
+        data = self.env['mail.channel'].read_group([('livechat_channel_id', 'in', self._ids), ('channel_message_ids', '!=', False)], ['__count'], ['livechat_channel_id'], lazy=False)
```

Comment:
The line was long before, but now it's just too long.
Can you please split with 1 line for each filter?